src/transformers/models/wav2vec2/tokenization_wav2vec2.py

```diff
@@ -234,7 +229,7 @@ def set_target_lang(self, target_lang: str):
         # characters are not split at tokenization
         for token in self.encoder.keys():
             if len(token) > 1:
-                self.unique_no_split_tokens.append(token)
+                self.add_tokens(token)
```

Comment:
`add_tokens` loops over the tokens, so I'd recommend adding them using:
```suggestion
        self.add_tokens(self.encoder.keys())
```
as an internal checks makes sure `let(token)>1`

Reply:
But we need `lstrip=rstrip=True` here? So need to use `AddedToken(lstrip=True, rstrip=True)` as we had before?